__tests__/app.test.js

```diff
@@ -49,60 +50,52 @@ describe("/api", () => {
             .then((result) => {
-                expect(Object.keys(result.body.endpoints).length).toBe(3);
+              expect(result.body.endpoints).toEqual(endpoints);
+              
             });
     });
-    test("json object should have a description key for each endpoint", () => {
-        return request(app)
-            .get("/api")
-            .expect(200)
-            .then((result) => {
-                expect(result.body.endpoints["GET /api"].description).toBe(
-                    "serves up a json representation of all the available endpoints of the api"
-                );
-                expect(result.body.endpoints["GET /api/topics"].description).toBe(
-                    "serves an array of all topics"
-                );
-                expect(result.body.endpoints["GET /api/articles"].description).toBe(
-                    "serves an array of all topics"
-                );
-            });
-    });
-    test("json object details the queries on each endpoint", () => {
-        return request(app)
-            .get("/api")
-            .expect(200)
-            .then((result) => {
-                expect(result.body.endpoints["GET /api/topics"].queries).toEqual([]);
-                expect(result.body.endpoints["GET /api/articles"].queries).toEqual([
-                    "author",
-                    "topic",
-                    "sort_by",
-                    "order",
-                ]);
-            });
-    });
-    test('json object should have an example response property for each endpoint', () => {
-        return request(app)
-            .get("/api")
-            .expect(200)
-            .then((result) => {
-                expect(result.body.endpoints["GET /api/topics"].exampleResponse).toEqual({
-                    "topics": [{ "slug": "football", "description": "Footie!" }]
-                });
-                expect(result.body.endpoints["GET /api/articles"].exampleResponse).toEqual({
-                    "articles": [
-                        {
-                            "title": "Seafood substitutions are increasing",
-                            "topic": "cooking",
-                            "author": "weegembump",
-                            "body": "Text from the article..",
-                            "created_at": "2018-05-30T15:59:13.341Z",
-                            "votes": 0,
-                            "comment_count": 6
-                        }
-                    ]
-                });
-            })
-    });
+
 })    
+
+describe("/api/articles/:article_id", () => {
+  test("GET - status 200 - respond swith a article of the corresponding artivcle id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((result) => { 
+          expect(typeof result.body.article.author).toBe("string");
+        expect(typeof result.body.article.title).toBe("string");
+        expect(typeof result.body.article.article_id).toBe("number");
```

Comment:
For article Id we only ever want the ID to be 1 here, so I would specifically check the article ID, the others however can remain typeof :)